Installers/Ubuntu-Install.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+echo Please enter your password to continue...
+
+# Install the following dependencies from repositories without asking questions, git-core can be removed afterwords as it's only used to initialise a local git repository
+echo Installing dependencies
+sudo apt-get -y install git-core libusb-1.0.0 libudev-dev python3 cython3 python3-pip python3-pil python3-setuptools
+
+cd /tmp
+pip3 install hidapi click
+
+# Get the sudoers home directory before changing directory into it
+cd ~
+
+# If the git repository already exists then delete it
+if [ -d "python-evic" ]; then
+  rm -Rf python-evic
```

Comment:
I second. An installer touching an existing (possibly dirty) tree is bad. By the way, wouldn't an user need to have already cloned the repo to have this script anyway?
I'd check 'git config --get remote.origin.url'. If the user cloned, just install. If not, it means the user downloaded the script directly from Github, so `mktemp -d` and clone since most likely he isn't interested in pulling updates.